message_sender/models.py

```diff
@@ -26,6 +26,7 @@ class Outbound(models.Model):
         "been answered. Not used for text messages")
     attempts = models.IntegerField(default=0)
     metadata = JSONField()
+    channel = models.CharField(null=True, blank=True, max_length=64)
```

Comment:
I made it a char because it will be created from other places that doesn't know what the ID is, like the SBM, are you suggesting a lookup before saving?